src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -111,8 +118,8 @@ class BaseRequestHandler(RequestHandler):
 
         class TornadoInstance(BaseRequestHandler):
 
-          # Prefix of methods names
-          METHOD_PREFIX = "export_"
+          # Prefix of methods names if need to use a special prefix. By default its "export_".
+          METHOD_PREFIX = "web_"
```

Comment:
This is confusing IMHO: is the default `export_` or `web_`?

Reply:
I'm very confused too...

If things are changing here can it just use a decorator instead of relying on magical method prefixes?

Reply:
`export_` is by default, `web_` default for REST/WebApp.